lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -1,4 +1,42 @@
 -- Propcore is allowed to everyone, but functions in the restrictedFunctions array will be restricted to devotee+ only
+local file_name = "cfc_propcore_whitelist"
```

Comment:
maybe a name such as `WHITELIST_FILEPATH` or `PROPCORE_WHITELIST_PATH` might be better

(you may even want to consider using the database)

Reply:
Yeah, I was thinking the same thing I just don't know a thing about interacting with the database...

Reply:
file-based seems fine to me actually. this won't be used enough for it to be a performance concern, and it lets us more easily audit the data